<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matrixrechner</class>
 <widget class="QWidget" name="Matrixrechner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrizrechner</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QWidget" name="wid_matA" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>311</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="wid_matB" native="true">
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>10</y>
           <width>311</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_AMalB">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>80</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>A*B</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>300</y>
           <width>1031</width>
           <height>121</height>
          </rect>
         </property>
         <widget class="QPushButton" name="btn_MatAP">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_MatAM">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>10</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_MatBP">
          <property name="geometry">
           <rect>
            <x>810</x>
            <y>0</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_MatBM">
          <property name="geometry">
           <rect>
            <x>880</x>
            <y>0</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_detA">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Determinate</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_mulitA">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Multiplicate with</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_multiA">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>70</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_powA">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>40</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Pow with</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_powA">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>40</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_rank">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>70</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Rank</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_trans">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>100</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Transponieren</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_kehr">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>100</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Kehrmatrix</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_kehrB">
          <property name="geometry">
           <rect>
            <x>870</x>
            <y>90</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Kehrmatrix</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_rankB">
          <property name="geometry">
           <rect>
            <x>870</x>
            <y>60</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Rank</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_mulitB">
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>60</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Multiplicate with</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_transB">
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>90</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Transponieren</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_detB">
          <property name="geometry">
           <rect>
            <x>750</x>
            <y>30</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Determinate</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_powB">
          <property name="geometry">
           <rect>
            <x>870</x>
            <y>30</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Pow with</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_multiB">
          <property name="geometry">
           <rect>
            <x>840</x>
            <y>60</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_powB">
          <property name="geometry">
           <rect>
            <x>960</x>
            <y>30</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="btn_APlusB">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>110</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>A+B</string>
         </property>
        </widget>
        <widget class="QWidget" name="wid_matC" native="true">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>430</y>
           <width>321</width>
           <height>251</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_AMinB">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>140</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>A-B</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_detA">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>430</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_detB">
         <property name="geometry">
          <rect>
           <x>770</x>
           <y>430</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_error">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>10</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
